recipes-devtools/clang/clang/llvm-config

```diff
@@ -8,7 +8,11 @@
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
 NEXT_LLVM_CONFIG="$(which -a llvm-config | sed -n 2p)"
 export YOCTO_ALTERNATE_EXE_PATH="${YOCTO_ALTERNATE_EXE_PATH:="$(readlink -f "$SCRIPT_DIR/../llvm-config")"}"
-export YOCTO_ALTERNATE_LIBDIR="${YOCTO_ALTERNATE_LIBDIR:="/lib"}"
+if [[ "$base_libdir" == *"lib64"* ]]; then
```

Comment:
`base_libdir` with `usrmerge` distro feature will prepend `/usr` to lib64, not that this patch is doing it but this will be a problem. Secondly, Using bashism needs to be removed as well.

Reply:
Yes, base_libdir consider usrmerge for prefix, but it also read value from baselib. That's why I am checking only for lib64 substring in base_libdir value. If I understood correctly !

From bitbake.conf:

root_prefix = "${@bb.utils.contains('DISTRO_FEATURES', 'usrmerge', '${exec_prefix}', '${base_prefix}', d)}"
export base_libdir = "${root_prefix}/${baselib}"

Reply:
btw. dont we need it in master too ? if so please post a pull for master as well.